cli/standalone.rs

```diff
@@ -109,6 +109,16 @@ fn read_string_slice(
 
 const SPECIFIER: &str = "file://$deno$/bundle.js";
 
+fn module_supported(specifier: &str) -> Result<(), AnyError> {
```

Comment:
This should be much more like the logic in `file_fetcher.rs`, like: https://github.com/denoland/deno/blob/main/cli/file_fetcher.rs#L153 and actually hopefully leveraging the same logic, just making it `pub(crate)` as needed, especially detecting if it is a valid data URL.

Reply:
`file_fetcher.rs` pulls with itself a lot of extra dependencies that `main_runtime` won't ever need. 
```diff
+ mod file_fetcher;
+ mod text_encoding;
+ mod media_type;
+ mod http_util;
+ mod auth_tokens;
+ mod http_cache;
+ mod fs_util;
+ mod checksum;
```
Maybe we can just have a stripped down version of `file_fetcher::get_source_from_data_url` for only validating data URL and returning the source.

Reply:
Ended up making a shared `mod data_url` between _cli_ and _denort_.